WordPress/gutenberg
# Remove PHPUnit and Composer packages

## What?
I have removed the `composer` and `phpunit` Docker services in favor of globally installing `composer` and `phpunit` in the `wordpress`, `cli`, `tests-wordpress`, and `tests-cli` containers.

## Why?
The current approach comes with a few compatibility issues:

- `phpunit`
  - [WordPress Core has stopped using the container](https://core.trac.wordpress.org/ticket/54112)
  - [It does not provide an ARM image](https://hub.docker.com/r/wordpressdevelop/phpunit)
- `composer`
  - It uses whatever version of PHP is packaged with the image when it is downloaded, not the version of PHP you have installed. This can lead to it installing incompatible PHP dependencies in certain circumstances.

## How?
This pull request removes the `composer` and `phpunit` services and installs the respective commands globally. In place of `npx wp-env run composer` you can now run `npx wp-env run cli composer`. One caveat is that the `composer` service used to default to the directory containing the config file. Instead, since it defaults to `/var/www/html`, they will need to run the command from the correct directory manually.

As an aside, I also got `sudo` installed in the CLI container so you have root access available in all of the containers. This is potentially useful for people who might install additional services or packages or what have you in a setup script.

## Testing Instructions

1. Run `npx wp-env destroy`.
2. Run `npx wp-env start --update`.
3. Run `npx wp-env run cli bash`.
4. Check out `composer` and `phpunit` and confirm they run.
5. Do the same in the `wordpress`, `tests-wordpress`, and `tests-cli` containers.
6. Run `npm run format:php` and confirm that it works as expected.
7. Run `npm run lint:php` and confirm that it works as expected.
